api/src/main/kotlin/io/spine/protodata/settings/Formats.kt

```diff
@@ -54,11 +53,11 @@ public val Format.extensions: List<String>
 /**
  * Obtains a [Format] from the file extension of the given configuration file.
  *
- * @throws ConfigurationError if the format is not recognized
+ * @throws IllegalStateException if the format is not recognized
```

Comment:
I am not sure how we agreed on dealing with `@throws` and capitalization. If we went for the simple rule, then it should be "If the format..." (and similar to the capitalized version you have in `Parser.kt`).

If we agreed on not capitalizing `@throws`, then `Parser.kt` has this issue, because in it, it is "If..."